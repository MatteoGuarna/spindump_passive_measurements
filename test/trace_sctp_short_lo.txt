A short test where SCTP communicates with itslef:
- one assoc is established
- one data in both directions
- HBs in both directions
- graceful shutdown
