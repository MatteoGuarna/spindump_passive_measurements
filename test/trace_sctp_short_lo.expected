SCTP 10.0.2.4 <-> 10.0.2.4 752068035:0 (37164:62324) at 1577108139309579 new starting packets 1 0 bytes 68 0 bandwidth 68 0
SCTP 10.0.2.4 <-> 10.0.2.4 752068035:281270461 (37164:62324) at 1577108139309793 measurement up right 11 packets 3 2 bytes 396 328 bandwidth 396 328
SCTP 10.0.2.4 <-> 10.0.2.4 752068035:281270461 (37164:62324) at 1577108139309903 measurement up left 72 packets 3 4 bytes 396 440 bandwidth 396 440
SCTP 10.0.2.4 <-> 10.0.2.4 752068035:281270461 (37164:62324) at 1577108174230168 measurement up right 118 packets 5 5 bytes 528 524 bandwidth 444 440
SCTP 10.0.2.4 <-> 10.0.2.4 752068035:281270461 (37164:62324) at 1577108174230173 measurement up left 16 packets 5 6 bytes 528 608 bandwidth 444 440
